srcpkgs/dinit/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'dinit'
+pkgname=dinit
+version=0.16.1
+revision=1
+build_style=gnu-makefile
+make_install_args="SBINDIR=/usr/bin"
+make_check_target="check"
+make_check_args="-j 1"  # building checks fails when run in parallel
+hostmakedepends="m4"
+checkdepends="libsanitizer-devel"
+short_desc="Service manager and init system"
+maintainer="Uli Baum <xeji@cat3.de>"
+license="Apache-2.0"
+homepage="https://davmac.org/projects/dinit/"
+changelog="https://github.com/davmac314/dinit/NEWS"
+distfiles="https://github.com/davmac314/dinit/archive/v${version}.tar.gz"
+checksum=fecab916da569e7fda1ad7651eb2cc639885ac203f4729d55612108b69cd5140
+
+post_extract() {
```

Comment:
this should be in post_patch or pre_configure

Reply:
Thanks for your super quick and clear feedback. I'm still learning :-)